Pipfile

```diff
@@ -29,8 +24,8 @@ tini = "*"
 "urllib3" = "*"
 whitenoise = "*"
 python-dateutil = "*"
-
+django-filters = "*"
```

Comment:
This needs to be `django-filter` (without the trailing `s`) because `django-filters` is not actually the package we need (otherwise the loading of `django_filters` in `settings.py` fails. I know, it's confusing.